src/main/java/smokefree/domain/RecordPlaygroundObservationCommand.java

```diff
@@ -13,6 +17,10 @@
     String initiativeId;
     String observer;
     Boolean smokefree;
+
+    @NotBlank(message = "Message must have at least " + SmokefreeConstants.PlaygroundObservation.MINIMUM_COMMENT_LENGTH + " character")
```

Comment:
drop this one

Reply:
I removed  it

Reply:
Revoed it in the other PR and rebase will solve the problem